test/basics.jl

```diff
@@ -0,0 +1,60 @@
+test_string_length = 100
+
+#string_types = [ASCIIStr, LatinStr, UTF8Str, UCS2Str, UTF16Str, UTF32Str, UniStr]
+#string_types = [ASCIIStr, LatinStr, UTF8Str, UCS2Str, UTF16Str, UTF32Str]
+string_types = [ASCIIStr, LatinStr]
+
+##  create type specific test strings
+test_strings_base = Dict()
+for string_type in string_types
```

Comment:
Follow the convention of T for a type variable where it's clear what it means

Reply:
OK, will do